source-browser.html

```diff
@@ -182,6 +193,29 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
       }
 
       $(document).ready(() => {
+        $(window).on('scroll resize', function(){
+            const top = $(window).scrollTop();
```

Comment:
The rest of the code is indented two spaces but now it's 4?

Reply:
I'll run it through Prettier again, then.